Makefile

```diff
@@ -113,3 +140,6 @@ brcm-all: target/sonic-generic.bin $(addprefix target/,docker-syncd.gz docker-or
 
 ## Note: docker-fpm.gz must be the last to build the implicit dependency fpmsyncd
 mlnx-all: target/sonic-generic.bin $(addprefix target/,docker-syncd-mlnx.gz docker-orchagent-mlnx.gz docker-fpm.gz docker-database.gz)
+
+## Note: docker-fpm.gz must be the last to build the implicit dependency fpmsyncd
+cavm-all: $(addprefix target/,docker-syncd-cavm.gz docker-orchagent-cavm.gz docker-fpm.gz docker-database.gz)
```

Comment:
We are adding it in different PR - https://github.com/Azure/sonic-buildimage/pull/4